test/e2e/specs/help-center/help-center__general-interaction.ts

```diff
@@ -2,11 +2,141 @@
  * @group calypso-pr
  */
 
-import { SupportComponent, TestAccount } from '@automattic/calypso-e2e';
+import {
+	DataHelper,
+	SupportComponent,
+	TestAccount,
+	TestAccountName,
+} from '@automattic/calypso-e2e';
 import { Browser, Page } from 'playwright';
 
 declare const browser: Browser;
 
+/** Tests to ensure the Help Center is open and visible in Calypso and the Editor */
+describe.each( [
+	{ accountName: 'defaultUser' as TestAccountName },
+	{ accountName: 'atomicUser' as TestAccountName },
+] )( 'Help Center: Verify Help Center is accessible', function ( { accountName } ) {
+	let page: Page;
+	let supportComponent: SupportComponent;
+	let testAccount: TestAccount;
+	const helpCenterContainerVisibilityErrorMessage = `This is a bug that should be urgently fixed.
+	But because this test runs against ETK production, this bug was probably not introduced in this pull request.
+	Please consider alerting the last person who deployed ETK to attend to this issue and fix the Help Center.`;
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page, { waitUntilStable: true } );
+
+		supportComponent = new SupportComponent( page );
+	} );
+
+	afterAll( async function () {
+		await page.close();
+	} );
+
+	describe( 'Verify Help Center is opened and visible in Calypso', function () {
+		it( 'Verify Help Center is initially closed', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeFalsy();
+		} );
+
+		it( 'Open Help Center', async function () {
+			await supportComponent.openPopover();
+		} );
+
+		it( 'Verify Help Center is opened', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeTruthy();
+		} );
+	} );
+
+	describe( 'Verify Help Center is opened and visible in Editor', function () {
+		it( 'Navigate to the Editor and verify the Help Center is initially closed', async function () {
+			const postURL = DataHelper.getCalypsoURL(
+				'/post/' + testAccount.getSiteURL( { protocol: false } )
+			);
+
+			await page.goto( postURL, {
+				waitUntil: 'networkidle',
+			} );
+		} );
+
+		it( 'Verify Help Center is initially closed', async function () {
+			expect( await page.locator( '.help-center__container' ).isVisible() ).toBeFalsy();
```

Comment:
This is looking for the Help Center in Calypso, not inside the iframe. You probably need

```js
const helpCenterContainerIframe = page
				.frameLocator( '.calypsoify iframe' )
				.locator( '.help-center__container' );`
```

Generally speaking, when testing a transition (A → B), you test A, make the transition, and the test B. The test should be as identical as possible and the only difference be what the transition is about.

So the `Verify Help Center is initially closed` test should be identical to `Verify Help Center is opened` but it should test for `falsy`. This way, your test would be solid as a rock.